Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,78 @@
 ﻿using System;
+using BootCamp.Chapter.Logger;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Print.Log.Warn("Program started.");
+            CalculateBmi();
+            Print.Log.Warn("Program finished.");
+        }
+
+        private static void CalculateBmi()
```

Comment:
instead of having raw code all in one place, split unrelated parts into functions, like you did in homework 3 of chapter 1.